README.md

```diff
@@ -324,6 +324,34 @@ let view3 = NibBasedRootView.loadFromNib() // and another one
 …
 ```
 
+## 4. Instantiating a `NibLoadable` UIViewController
+
+`Reusable` also allows you to mark your `UIViewController` classes as `NibLoadable` to easily instantiate them from their associated XIB in a type-safe way.
+
+## 4a. Declare your `UIViewController` to conform to `NibLoadable` 
+
+In your swift source declaring your custom `UIViewController` class:
+
+* Use the `NibLoadable` protocol if you used a "File's Owner" of the XIB being of the class of your custom view controller.
+
+```swift
+// a XIB-based custom UIViewController, used as the XIB's "File's Owner"
+final class CustomViewController: UIViewController, NibLoadable { /* and that's it! */ }
+
+```
+
+## 4b. Instantiate your UIViewControllers
+
+Simply call `instantiate()` on your custom class. This will automatically know the file `*.xib` to load and instantiate it.
+
+```swift
+func presentCustomViewControllerFromXib() {
+  let vc = CustomViewController.instantiate() // Init from the "CustomViewController" of CustomViewController.xib
```

Comment:
Tbh now that I see the full implementation and usage I'm not sure I see the benefit of having all this in Reusable? I mean what's the difference between making `CustomViewController` conform to `NibLoadable` then call `CustomViewController.instantiate()` vs not having any of all this and just calling `CustomViewController()` directly, without even having the need for Reusable in the first place for this use case?

I mean, in case you didn't know and if I'm not mistaken, since a while (iOS 6 or something?), the implementation of `init()` on `UIViewController` automatically forwards to `init(nibName:bundle:)` if iOS finds a XIB with the same name as the class. So all this dance is already implementation by the OS for us…